### DISCUSSION TITLE
Rearrange the model versions

### DISCUSSION THREAD

**Discussion by ostap667inbox** [civitai/civitai#431] [Ideas]
It would be nice to be able to rearrange the model versions. Seen it from other users and ran into a similar problem myself yesterday:

First downloaded the main version of the model, and then the idea arose to download the previous intermediate version, because for some tasks it is better suited. 
And in the list of versions the older version is now in first place. Also by default when you open the model page you see exactly this downloaded last version, this is wrong. 
In addition, after the recent changes to the site, it is difficult for users to pay attention to the list with a list of versions and that they are not in order of number. Very inconvenient to use it.
![00005](https://user-images.githubusercontent.com/123593542/230567321-50e04c99-9299-417e-ad3e-74bbe87acfda.png)

**Comment by JustMaier** [comment]
You can reorder versions from the model context menu in the top right of your model page:
![image](https://user-images.githubusercontent.com/607609/231455857-ae77afa0-bdfc-4764-9043-e2f6acdcf150.png)
![image](https://user-images.githubusercontent.com/607609/231455932-f180ee0e-8df1-436f-b994-c7a0f9527f58.png)

It's not very obvious. I'll work with the team to add something a little more obvious

**Reply by ostap667inbox** [reply]
Thank you. The new way looks more obvious and understandable:
![chrome_DGds68J2ig](https://user-images.githubusercontent.com/123593542/231671795-390df645-251c-4e52-ab97-08efc6464d20.png)

**Reply by JustMaier** [reply]
Awesome. Glad to hear you like it.
Thanks again for the feedback 👍
